test/videojs-contrib-hls.test.js

```diff
@@ -2072,6 +2061,54 @@ QUnit.test('resolves relative key URLs against the playlist', function() {
               'resolves the key URL');
 });
 
+QUnit.test('adds 1 empty audio track if we have not parsed any, and the playlist is loaded', function() {
```

Comment:
It will look like this: 
```JavaScript
  {
    language: '',
    label: '',
    enabled: true,
    id: '1',
    kind: 'main',
    tech: this.tech_
}));
``` JavaScript